CHANGELOG.md

```diff
@@ -1,3 +1,165 @@
+# Kubermatic 2.16
+
+## [v2.16.0-beta.1](https://github.com/kubermatic/kubermatic/releases/tag/v2.16.0-beta.1)
+
+Before upgrading, make sure to read the [general upgrade guidelines](https://docs.kubermatic.com/kubermatic/v2.16/upgrading/guidelines/)
+as well as the [2.16 upgrade notes](https://docs.kubermatic.com/kubermatic/v2.16/upgrading/2.15_to_2.16/).
+
+### Supported Kubernetes Versions
+
+* 1.17.9
+* 1.17.11
+* 1.17.12
+* 1.17.13
+* 1.17.16
```

Comment:
Is it expected that we still support 1.17 since it has reached EOL in mid-January?